eva-server/src/test/java/uk/ac/ebi/eva/server/ws/VariantWSServerTest.java

```diff
@@ -172,4 +172,22 @@ private Boolean testCheckVariantExistsHelper(String testIdRegion) throws URISynt
         return results.get(0);
     }
 
+    @Test
+    public void testCountTotalNumberOfVariantsService() throws URISyntaxException {
```

Comment:
The method name should be focused on the functionality more than architecture details so I think `testCountTotalNumberOfVariants` would be a better name.

This comment also applies to the tests in `VariantWSServerIntegrationTest`.